hands-on/partials/_next.html.md

```diff
@@ -7,6 +7,7 @@ If you're all about the code, then these guides will get you deploying from your
 * [Learn how to build and deploy the hellofly application from Go source](/docs/getting-started/golang/)
 * [Deploying a Node application with Fly.io](/docs/getting-started/node/)
 * [Deploying a Deno application with Fly.io](/docs/getting-started/deno/)
+* [Deploying a Laravel application with Fly.io](/docs/getting-started/laravel/)
```

Comment:
This redirects to `/docs/laravel/` - The middleman code is sometimes brittle, so let's use the final URL `/docs/laravel/` here.

I suppose we could be fair to our Rails friends and add them in here too 🤔 

```
* [Deploying a Rails application with Fly.io](/docs/rails/)
```

Reply:
Sure thing!